### PR TITLE
Corrects blocked count colors

### PR DESCRIPTION
**Pull request by Jacalz**
This makes the colors in the shield panel the same as on browser-laptop :+1: 
fixes: #932

### CODE REVIEW

File: brave/src/frontend/sidepanels/BraveRightSidePanel.swift [hunk 1]
```diff
@@ -375,9 +375,9 @@ class BraveRightSidePanelViewController : SidePanelBaseViewController {
             let statTitles = [Strings.Ads_and_Trackers, Strings.HTTPS_Upgrades, Strings.Scripts_Blocked, Strings.Fingerprinting_Methods]
             let statViews = [statAdsBlocked, statHttpsUpgrades, statScriptsBlocked, statFPBlocked]
             let statColors = [UIColor(red:234/255.0, green:90/255.0, blue:45/255.0, alpha:1),
-                              UIColor(red:242/255.0, green:142/255.0, blue:45/255.0, alpha:1),
                               UIColor(red:26/255.0, green:152/255.0, blue:252/255.0, alpha:1),
-                              UIColor.init(white: 90/255.0, alpha: 1)]
+                              UIColor(white: 90/255.0, alpha: 1)]
```
Comment (jhreis) [inline brave/src/frontend/sidepanels/BraveRightSidePanel.swift]: Just remove trailing `]` and replace with `,`

File: brave/src/frontend/sidepanels/BraveRightSidePanel.swift [hunk 2]
```diff
@@ -375,9 +375,9 @@ class BraveRightSidePanelViewController : SidePanelBaseViewController {
...
+                              UIColor.init(red:242/255.0, green:142/255.0, blue:45/255.0, alpha:1)]
```
Comment (jhreis) [inline brave/src/frontend/sidepanels/BraveRightSidePanel.swift]: Remove `.init` call, use direct constructor.

**Comment by Jacalz** [comment]
Will fix the requested changes and also fix a thing that I did wrong 😉 I only switched the colors around but browser-laptop uses a yellow color and not orange 🙂

**Comment by jhreis** [comment]
@Jacalz Yes, please make sure to use the exact same colors.

File: brave/src/frontend/sidepanels/BraveRightSidePanel.swift (hunk 1 above)
Comment (Jacalz) [inline brave/src/frontend/sidepanels/BraveRightSidePanel.swift]: Fixed with https://github.com/brave/browser-ios/pull/933/commits/df18291f465e8626ffc87a83fafa89d37bde4480

File: brave/src/frontend/sidepanels/BraveRightSidePanel.swift (hunk 2 above)
Comment (Jacalz) [inline brave/src/frontend/sidepanels/BraveRightSidePanel.swift]: Fixed with https://github.com/brave/browser-ios/pull/933/commits/df18291f465e8626ffc87a83fafa89d37bde4480

**Comment by Jacalz** [comment]
I also made the total blocked colors be the same as browser-laptop and to make it correspond with the shields panel :+1: I accidentaly hit enter so that is why the lates commit message is a bit badly spelled :joy:

**Comment by Jacalz** [comment]
Closing this as github decided that there still is merge conflict even after fixing the conflicts, but I will open a new pr with some clean up :+1:

**Pull request closed without merging**